src/user/mod.rs

```diff
@@ -419,54 +391,3 @@ pub fn updates(req: &mut Request) -> CargoResult<Response> {
         meta: Meta { more: more },
     }))
 }
-
-#[cfg(test)]
```

Comment:
Actually, I believe the intent of the first test is covered under `create_token_multiple_have_different_values` in `src/test/token`. The second one I can try and come up with an equivalent with the new data structure.